02-hello-flutter/projects/final/recipes/lib/recipe_detail.dart

```diff
@@ -32,7 +32,8 @@ class _RecipeDetailState extends State<RecipeDetail> {
             SizedBox(
               height: 300,
               width: double.infinity,
-              child: Image(image: AssetImage(widget.recipe.imageUrl)),
+              child: Image(image:
+              AssetImage(widget.recipe.imageUrl)),
```

Comment:
@mikekatz Was this needed because of some rendering issue on web/ePub? 
I checked and Android Studio is not complaining for the line length of the old version.